libs/spaces/data/src/lib/store/spaces.store.ts

```diff
@@ -1,26 +1,28 @@
 import { Injectable } from '@angular/core';
-import { SettingsApi, SettingsType } from '@architect-poc/settings/public/interfaces';
+import {
+  SettingsSharedFeatureState,
+  SettingsType,
+} from '@architect-poc/settings-public-state';
 import { Observable, of } from 'rxjs';
 import { map } from 'rxjs/operators';
 import { Space } from '@architect-poc/spaces/domain';
-import { SpacesDataPort } from '@architect-poc/spaces/use-cases';
+import { SpacesFeatureState } from '@architect-poc/spaces/use-cases';
+import { SpacesResource } from '../resource/spaces.resource';
 
 @Injectable({ providedIn: 'root' })
-export class SpacesStoreService implements SpacesDataPort {
-
-
+export class SpacesStore implements SpacesFeatureState {
   getSpaces(): Observable<Space[]> {
-    return of(new Array(10).fill(0).map((_, i) => ({
-      id: `${i}`,
-      name: `space no ${i}`
-    })));
+    return this.spacesResource.getSpaces();
   }
 
-  constructor(private readonly settingsStore: SettingsApi) {}
+  constructor(
+    private readonly settingsStore: SettingsSharedFeatureState,
+    private readonly spacesResource: SpacesResource
```

Comment:
Not sure about the naming here as well. What about sticking to the commonly used `SpacesService` for the classes that are using the `HttpClient` for fetching the data from the server? But maybe this isn't that important at the moment (just trying to do the review properly 😄).

Reply:
michael wanted to name http services as `Resource`